cfme/containers/provider/openshift.py

```diff
@@ -20,10 +20,15 @@ class OpenshiftProvider(ContainersProvider):
     db_types = ["Openshift::ContainerManager"]
 
     def __init__(self, name=None, credentials=None, key=None,
-                 zone=None, hostname=None, port=None, provider_data=None, appliance=None):
+                 zone=None, hostname=None, port=None, provider_data=None, appliance=None,
+                 sec_protocol=None):
         super(OpenshiftProvider, self).__init__(
             name=name, credentials=credentials, key=key, zone=zone, hostname=hostname, port=port,
-            provider_data=provider_data, appliance=appliance)
+            provider_data=provider_data, appliance=appliance, sec_protocol=sec_protocol)
+
+        # Here drop a selector of OSE versions?
+        if self.appliance.version.vstring > '5.8.0.3':
+            self.sec_protocol = sec_protocol
```

Comment:
Hmmm I tried something like this:
    def _form_mapping(self, create=None, **kwargs):
        if self.appliance.version > '5.8.0.3':
            sec_protocol = kwargs.get('sec_protocol')
        else:
            sec_protocol = None

        return {'name_text': kwargs.get('name'),
                'type_select': create and 'OpenShift',
                'hostname_text': kwargs.get('hostname'),
                'port_text': kwargs.get('port'),
                'sec_protocol': sec_protocol,
                'zone_select': kwargs.get('zone')}
I also can't leave self.sec_protocol = sec_protocol up there if I remove it from the _init_ and from the Super()
But it doesn't work..